### DISCUSSION TITLE
Rotated shape is drawn without antialiasing

### DISCUSSION THREAD

**Discussion by sferhah** [taublast/DrawnUi#121] [Q&A]
Screenshot (top is with DrawnUi.Maui controls, bottom is with Maui controls)

<img width="314" alt="skia_vs_maui" src="https://github.com/user-attachments/assets/22c00d90-bf54-47a3-8c8a-1b29bb44346a">

Notice that there are no visibile pixels with the Maui Line control.

Here's my code:

```
    private static (SkiaShape, Point) CreateSkiaLine(Color color, double x1, double y1, double x2, double y2, int width = 5)
    {
        var distX = x1 - x2;
        var distY = y1 - y2;
        var lineDist = Math.Sqrt(distX * distX + distY * distY);

        var lineCenterX = x2 + (distX / 2);
        var lineCenterY = y2 + (distY / 2);
        var lineFinalX = lineCenterX - (lineDist / 2);

        var lineRotateAngle = Math.Acos(distX / lineDist) * (180 / Math.PI);

        var line = new SkiaShape()
        {            
            Type = ShapeType.Rectangle,
            BackgroundColor = color,
            WidthRequest = lineDist,
            HeightRequest = width,
            Rotation = (distY < 0) ? 360 - lineRotateAngle : lineRotateAngle
        };

        return (line, new Point(lineFinalX, lineCenterY));
    }
```

**Comment by taublast** [comment]
Hi what nuget version are you using please and on what platform? Thanks.

Have my Iill' idea about it already..

**Reply by sferhah** [reply]
Nuget: version 1.2.5.2, Platform: WinUI.

Here's a screenshot on Android (top is DrawnUi, bottom is Maui):

<img width="309" alt="skia_vs_maui_android" src="https://github.com/user-attachments/assets/51b09141-b8ee-45e5-85fe-5c33e71c734a">

It's less pixelated, but the Maui Line is still "smoother".

**Reply by taublast** [reply]
Okay thanks! Could investigate only tomorow, thanks for reporting will move this to issues! 
We have clearly an issue here with the matrix transforms rendering:  paint antialiasing not being applied.

**Comment by taublast** [comment]
Opened issue https://github.com/taublast/DrawnUi.Maui/issues/122
